### PR TITLE
Add Mere-exposure

### PR DESCRIPTION
**Pull request by HariMahi**

### CODE REVIEW

**Comment by HariMahi** [comment]
Please review this PR

File: readme.md [hunk 1]
```diff
@@ -176,6 +179,37 @@ the anecdote, you can throw away the data."
...
+
+[R: health.harvard.edu](https://www.health.harvard.edu/mental-health/the-power-of-the-placebo-effect)
+
+[R: betterhealth.vic.gov.au](https://www.betterhealth.vic.gov.au/health/conditionsandtreatments/placebo-effect)
+
+#### [Mere-exposure Effect](https://en.wikipedia.org/wiki/Mere-exposure_effect)
+
+> Repeated exposure of something leads to a more positive feeling about it.
+
+The mere-exposure effect is a psychological phenomenon by which people tend to develop a preference for things merely because they are familiar with them. In social psychology, this effect is sometimes called the familiarity principle. The effect has been demonstrated with many kinds of things, including words, Chinese characters, paintings, pictures of faces, geometric figures, and sounds. In studies of interpersonal attraction, the more often a person is seen by someone, the more pleasing and likeable that person appears to be.
```
Comment (icyflame) [inline readme.md]: wrap at 70

File: readme.md [hunk 2]
```diff
@@ -176,6 +179,37 @@ the anecdote, you can throw away the data."
...
+
+[R: psychologytoday.com](https://www.psychologytoday.com/blog/ulterior-motives/200811/know-me-is-me-i-mere-exposure)
+
+#### [Boanthropy](https://en.wikipedia.org/wiki/Boanthropy)
+
+> Human believes himself or herself to be a bovine.
+
+Boanthropy is a strange delusional disorder during which a person imagines him- or herself to be a cow or a bull. The disorder usually begins as a sort of dream, and then takes over a person entirely, eventually becoming a mania. It is also possible that this state can be caused by hypnosis, provided that a person is susceptible to it.
+
+[R: psychologyart.wordpress.com](https://psychologyart.wordpress.com/2016/03/23/boanthrophy/)
```
Comment (icyflame) [inline readme.md]: add atleast 1 more ref

File: readme.md [hunk 3]
```diff
@@ -176,6 +179,37 @@ the anecdote, you can throw away the data."
...
+
+[R: socialpsychonline.com](http://socialpsychonline.com/2016/03/the-mere-exposure-effect/)
+
+[R: psychologytoday.com](https://www.psychologytoday.com/blog/ulterior-motives/200811/know-me-is-me-i-mere-exposure)
+
+#### [Boanthropy](https://en.wikipedia.org/wiki/Boanthropy)
+
+> Human believes himself or herself to be a bovine.
+
+Boanthropy is a strange delusional disorder during which a person imagines him- or herself to be a cow or a bull. The disorder usually begins as a sort of dream, and then takes over a person entirely, eventually becoming a mania. It is also possible that this state can be caused by hypnosis, provided that a person is susceptible to it.
```
Comment (icyflame) [inline readme.md]: wrap at 70

File: readme.md [hunk 4]
```diff
@@ -176,6 +179,37 @@ the anecdote, you can throw away the data."
 
 [R: verywell.com](https://www.verywell.com/what-is-the-hawthorne-effect-2795234)
 
+#### [Placebo Effect](https://en.wikipedia.org/wiki/Placebo)
```
Comment (icyflame) [inline readme.md:181]: this is already in master. It shouldn't be a part of your patch. 

Solution: Rebase your branch onto the latest master to clear this up.

**Review by icyflame — CHANGES_REQUESTED** [review]

**Comment by HariMahi** [comment]
Please review #43 and #44 PRs. I have made the changes.

**Pull request closed without merging**